### PR TITLE
gay

### PR DESCRIPTION
**Pull request by FactAttacker**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,5 @@ typings/
 
 # next.js build output
 .next
+
+#태성씨 게이
```
Comment (FactAttacker) [inline .gitignore:63]: true

Comment (geoseong) [reply .gitignore:63]: @FactAttacker i'm not gay

Comment (geoseong) [reply .gitignore:63]: fix this right now

**Review by geoseong — CHANGES_REQUESTED** [review]
취소하슈

**Review by FactAttacker — COMMENTED** [review]
Test

**Pull request closed without merging**